CMakeLists.txt

```diff
@@ -4,10 +4,11 @@ project(revpi-tools)
 
 include(GNUInstallDirs)
 
+add_subdirectory(avahi-services)
 add_subdirectory(examples-watchdog)
-add_subdirectory(udev)
-add_subdirectory(systemd)
-add_subdirectory(revpi-sos)
 add_subdirectory(fixup)
-add_subdirectory(tools)
 add_subdirectory(revpi-factory-reset)
+add_subdirectory(revpi-sos)
+add_subdirectory(systemd)
+add_subdirectory(tools)
+add_subdirectory(udev)
```

Comment:
You are reordering the lines. I get why you are doing this, but please keep refactoring changes separate from functional changes.